dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/config/MasterConfig.java

```diff
@@ -48,16 +46,8 @@ public class MasterConfig implements Validator {
      * The master RPC server listen port.
      */
     private int listenPort = 5678;
-    /**
-     * The thread number used to prepare processInstance. This number shouldn't bigger than fetchCommandNum.
-     */
-    private int preExecThreads = 10;
-    /**
-     * todo: We may need to split the process/task into different thread size.
-     * The thread number used to handle processInstance and task event.
-     * Will create two thread poll to execute {@link WorkflowExecuteRunnable} and {@link TaskExecuteRunnable}.
-     */
-    private int execThreads = 10;
+
+    private int workflowEventBusFireThreadCount = Runtime.getRuntime().availableProcessors() * 2 + 1;
```

Comment:
It's better to add these config to `application.yaml` and add comments in it to let the user know its meaning. Same as `masterSyncTaskExecutorThreadPoolSize` and `masterAsyncTaskExecutorThreadPoolSize`.

Is it too large for `Runtime.getRuntime().availableProcessors() * 2 + 1`? I think `Runtime.getRuntime().availableProcessors()` is enough.

Reply:
Done, added to application.yaml, the default value Runtime.getRuntime().availableProcessors() * 2 + 1 is not large, since there exists some DB operation in event firing.